plugins/inputs/infiniband/infiniband.go

```diff
@@ -0,0 +1,19 @@
+package infiniband
+
+// Stores the configuration values for the infiniband plugin - as there are no
+// config values, this is intentionally empty
+type Infiniband struct {
+}
+
+// Sample configuration for plugin
+var InfinibandConfig = `
+  ## no config required
```

Comment:
You can remove this comment line, Telegraf has magic to add something very similar when there is no config.

Reply:
Removed